transports/transports-deciding.md

```diff
@@ -0,0 +1,196 @@
+- Azure Storage Queues
+- SQL Server
+- RabbitMQ
+- Amazon SQS
+
+For every cloud hosted transport, it is possible that the network connection to the cloud provider might suffer. The result is that you cannot submit new messages and can possibly result in lost data from, for example, a user interface. If this user interface is also running at the same datacenter, this is less likely to occur.
+
+## Learning Transport
+
+This transport should not be used in production!
+
+It has benefits for learning how to work with NServiceBus. Instead of requiring additional technology installed, it will work straight out of the box. This is achieved by the fact that this transport stores messages as files on disk which can be read by other endpoints.
+
+
+
+## MSMQ Transport
+
+The MSMQ transport uses native Windows queuing mechanism, MSMQ, to deliver messages. MSMQ is a distributed system that consists of multiple processes, one on each machine. The client always interacts with a process running locally and the messages are forwarded in the background.
+
+#### Advantages
+
+- A built-in component of Windows operating system
+- Supports distributed transactions allowing atomic message receive and data manipulation through [Microsoft Distributed Transaction Coordinator (MSDTC)](https://msdn.microsoft.com/en-us/library/ms684146.aspx)
+- Provides a store and forward mechanism which allows endpoints to send messages even though the destination endpoint might be temporarily unreachable due to network issues.
+
+#### Disadvantages
+
+- Does not offer a native publish-subscribe mechanism, therefore it requires NServiceBus persistence to be configured for storing event subscriptions. [Explicit routing for publish/subscribe](/nservicebus/messaging/routing.md#event-routing-message-driven) must also be specified.
+- Many organizations don't have the same level of operational expertise with MSMQ that they do with other technologies (e.g. SQL Server), so it may require additional training.
+- Scaling out compute-intensive workload require setting up the distributor or sender-side distribution of messages to destination queues. This is harder than with the competing consumer pattern that other transports support.
+
+#### When would you select this transport
+
+- If you need store & forward for a higher reliability that your queuing technology is always available.
+- If you’re running a Windows environment on-premise and don’t want to spend additionally on licenses and/or training.
+- If you are able to coop distributed transactions and want the reliability it adds.
+
+
+
+## Azure Service Bus Transport
+
+Azure provides multiple messaging technologies and one of the most advanced and reliable ones is Azure Service Bus.
+
+Advantages
+
+- No need to maintain infrastructure or have deep knowledge of Azure Service Bus as Microsoft and NServiceBus take care of almost everything.
+- Ability to scale in both features and price range.
+- AMQP based implementation
```

Comment:
I can see how this is better over ASQ, that is HTTP-based.

Reply:
That is already listed as a disadvantage of ASQ. That is the correct place to list it, since it is a disadvantage of ASQ _compared to most of the other transports_. Whereas it is not an advantage of ASB _compared to most of the other transports_.

```suggestion
```